tutorial/buy-name.md

```diff
@@ -21,27 +21,27 @@ func NewMsgBuyName(name string, bid sdk.Coins, buyer sdk.AccAddress) MsgBuyName
 	}
 }
 
-// Type Implements Msg.
+// Route should return the name of the module
```

Comment:
If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.

Reply:
That does sound logical ;)